lib/report_portal/cucumber/parallel_report.rb

```diff
@@ -86,14 +78,21 @@ def done(desired_time = ReportPortal.now)
 
       private
 
-      def file_with_launch_id
-        Pathname(Dir.tmpdir) + "parallel_launch_id_for_#{@pid_of_parallel_tests}.lck"
+      def lock_file(file_path = nil)
+        file_path ||= Dir.tmpdir + "parallel_launch_id_for_#{@pid_of_parallel_tests}.lock"
+        super(file_path)
       end
 
       def set_parallel_tests_vars
         pid = Process.pid
         loop do
+          #FIXME failing in jenkins
           current_process = Sys::ProcTable.ps(pid)
+          #TODO: add exception to fall back to cucumber process 
```

Comment:
Sorry, but I don't understand that comment.

Reply:
Current implementation required to pass different formater when executing cucumber vs parallel_cucumber.
This is actually adding unnecessary complexity on test framework, basic idea is to make single formater that can work with cucumber and parallel_cucumber